modules/core/src/main/java/org/apache/ignite/IgniteJdbcDriver.java

```diff
@@ -130,139 +123,9 @@
  *         <p> Defaults to {@code false}, meaning that the whole result set is fetched to memory eagerly.
  *     </li>
  * </ul>
- *
```

Comment:
The javadoc above still mentions Java Client Based driver
`* JDBC driver can return two different types of connection: Ignite Java client based connection`
Let's remove it.